roles/apache/templates/vhost.conf.j2

```diff
@@ -1,14 +1,18 @@
 # {{ ansible_managed }}
 # Virtual host for {{ apache_server_name }}
 
+{% if vip_address is not defined %}
```

Comment:
Nice, I like this. I think a better name for it might be `apache_server_bind_address`, just to match the others; I never really liked how I re-used `external_vip_address`.